src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.NET.Sdk.DefaultItems.targets

```diff
@@ -22,6 +22,10 @@ Copyright (c) .NET Foundation. All rights reserved.
     <EnableDefaultNoneItems Condition=" '$(EnableDefaultNoneItems)' == '' ">true</EnableDefaultNoneItems>
   </PropertyGroup>
 
+  <ItemGroup>
+    <None Remove="@(Compile)" Condition=" '$(EnableDefaultCompileItems)' != 'true' And '$(EnableDefaultNoneItems)' == 'true' " />
```

Comment:
Sure.

The others `Remove` are working now, because it just add everything and next step remove it.

Yes, the 

```xml
<None Remove="**/*$(DefaultLanguageSourceExtension)" Condition=" '$(EnableDefaultCompileItems)' == 'true' "/>`
```

is pretty much useless afaik if i juse if 

```xml
<None Remove="@(Compile)" Condition=" '$(EnableDefaultNoneItems)' == 'true' " />
```

to remove all `Compile` from `None` (i think is never right to be both `None` and `Compile`, but dunno, i dont want to create regression).

just tell me @eerhardt , np for me